src/main/java/games/strategy/triplea/delegate/BidPlaceDelegate.java

```diff
@@ -82,11 +82,6 @@ protected int getMaxUnitsToBePlaced(final Collection<Unit> units, final Territor
     return units.size();
   }
 
-  // Allow player to place as many units as they want in bid phase
-  protected int getMaxUnitsToBePlaced(final Territory to, final PlayerID player) {
```

Comment:
Neither parameter of this method was used, but since the method itself appears to have no callers, I simply removed the whole thing.